DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+                              "Att": [
+                                {
+                                  "Id": "9a5247db-2c9e-4185-9991-45aa04a0b29d",
+                                  "Nm": "Repair Type conformation",
+                                  "Dsc": "Repair Typer Confirmation Description",
+                                  "AttTy": "UserEntry",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "isHidden": true,
+                                  "UICmpt": {
+                                    "Order": 2,
+                                    "CmptNm": "RepairType",
+                                    "DisplayTtl": "",
+                                    "Tooltip": "",
+                                    "CmptTy": "RadioButton",
+                                    "DisplayDsc": {
+                                      "DisplayTxt": ""
+                                    },
+                                    "ValTy": "Standard"
+                                  },
+                                  "Enum": [
+                                    {
+                                      "Val": "UnderWarranty",
+                                      "DisplayTxt": "Under Warranty",
+                                      "SrtOrd": 1,
+                                      "Nm": "Opt1"
+                                    },
+                                    {
+                                      "Val": "NoUnderWarranty",
+                                      "DisplayTxt": "Not Under Warranty",
+                                      "SrtOrd": 2,
+                                      "Nm": "Opt2"
+                                    }
+                                  ],
+                                  "Rul": [
+                                    {
+                                      "RulId": "607d8481-ba7c-4c95-ae07-8273cc077f59",
+                                      "Nm": "Rule condition to Repair Type",
+                                      "Cond": "**.Att[Id='75e0a595-ab28-4e91-9949-fab52cdf82b5'].Val='Opt1'",
+                                      "AttrMtdt": [
+                                        {
+                                          "isMan": true,
+                                          "isHidden": false
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "18b014c0-2dd2-4888-850b-e52e28e24657",
+                                  "Nm": "Purpose Name - Repair Type",
+                                  "Dsc": "Purpose Name - Repair Type desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "User does not have Repair Type",
```

Comment:
it sohuld be "Agree to User have Repair Type"

Reply:
Done